src/psyclone/parse/module_info.py

```diff
@@ -35,16 +35,23 @@
 
 '''This module contains the ModuleInfo class, which is used to store
 and cache information about a module: the filename, source code (if requested)
-and the fparser tree (if requested).
+and the fparser tree (if requested), and information about routine it
+includes, and external symbol usage.
 '''
 
+import os
 
 from fparser.common.readfortran import FortranStringReader
-from fparser.two.Fortran2003 import Use_Stmt
+from fparser.two.Fortran2003 import (Function_Subprogram, Interface_Block,
+                                     Interface_Stmt, Procedure_Stmt,
+                                     Subroutine_Subprogram, Use_Stmt)
 from fparser.two.parser import ParserFactory
-from fparser.two.utils import walk
+from fparser.two.utils import FortranSyntaxError, walk
 
-from psyclone.errors import PSycloneError
+from psyclone.errors import InternalError, PSycloneError
+from psyclone.psyir.nodes import Container, FileContainer, Routine
+from psyclone.psyir.frontend.fparser2 import Fparser2Reader
```

Comment:
Alpha-order please.

Reply:
I meant that the lines should be ordered by the modules they import from but I don't care that much :-)